lib/id.js

```diff
@@ -137,6 +137,16 @@ const idToPath = id => {
   return path;
 };
 
+const pathToId = path => {
+  const chunks = path.split('/');
+  let hex = '0x';
```

Comment:
You don't need to add `0x` at the start of the number string if you are using `parseInt()` providing the base to it.
Also, you can make this function a one-liner that way.

Reply:
Sorry, I already landed these PR. But yes, `split.reverse.join` may turn this func to one-lien. In spite of this such an implementation is good enough and may even be faster. @belochub

Reply:
@tshemsedinov, you've forgotten to land both commits or to squash commits before landing.

Reply:
Also, you can make it a one liner that way:
```js
parseInt(path.split('/').reduceRight((hex, chunk) => hex + chunk), 16);
```

Reply:
@belochub thanks, I'll add it as a separate commit